xm-commons-scheduler/src/main/java/com/icthh/xm/commons/scheduler/adapter/SchedulerChannelManager.java

```diff
@@ -64,7 +64,7 @@
 
 @Slf4j
 @Component
-@EnableBinding
+//@EnableBinding todo should be removed ??
```

Comment:
1 - pls check how queue metrics works without this annotation
2 - how about migrate to DynamicConsumer ?